docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1700 @@
+{
```

Comment:
<p>I wouldn't call this <code>noisy</code>. To me "noisy" implies that you have added some white noise to the time series which is not the case here.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>I tried to checkout other conversations/notebooks in STUMPY and, it seems there is no particular name for a time series with NaN/inf. So, I think I can simply just call it <code>T</code> again. Sounds good? </p>

Reply:
<p>Sure. I don't know if we really need to make a copy in this case? Are we planning to re-use the original <code>T</code>?  </p>